src/www/ui/api/documentation/openapi.yaml

```diff
@@ -866,7 +866,48 @@ paths:
                   $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
-
+  /uploads/{id}/licenses/{licenseId}/main:
+    parameters:
+      - name: id
+        required: true
+        description: Id of the upload
+        in: path
+        schema:
+          type: integer
+      - name: licenseId
+        required: true
+        description: Id of the main license to be deleted
+        in: path
+        schema:
+          type: integer
```

Comment:
Endpoint in PR #2462 uses shortname to add a main license and this endpoint uses a license id to remove it. This causes inconsistency between endpoints for same info.
I will recommend to create an endpoint to remove the main license in same way as PR #2462 .

Reply:
updated